### PR TITLE
Feature/add likes

### PR DESCRIPTION
**Pull request by nndd91**
Add likes functionality with ajax to refresh the likes.

### CODE REVIEW

File: spec/models/like_spec.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
 
 RSpec.describe Like, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
```
Comment (gohdaniel15) [inline spec/models/like_spec.rb]: Missing specs?

Comment (nndd91) [reply spec/models/like_spec.rb]: opps forgot about model specs. xD Will get to it right away

Comment (nndd91) [reply spec/models/like_spec.rb]: ok added validations and spec and pushed up :)

**Comment by nndd91** [comment]
fireshot capture 3 - twitclone - http___localhost_3000_
fireshot capture 4 - twitclone - http___localhost_3000_users_edit
fireshot capture 5 - twitclone - http___localhost_3000_profile_1
fireshot capture 6 - twitclone - http___localhost_3000_profile_1_get_following

Some screenshot

File: Gemfile.lock [hunk 2]
```diff
@@ -79,6 +79,8 @@ GEM
       i18n (~> 0.5)
```
Comment (ngerald51) [inline Gemfile.lock]: Please put this GemFile.lock in the gitignore list. thanks

**Review by ngerald51 — COMMENTED** [review]
Please add gemfile.lock in gitignore

File: Gemfile.lock (hunk 2 above)
Comment (nndd91) [reply Gemfile.lock]: Ok done. I realised its in our ignore file but I missed an asterisk that time.

**Pull request merged**